app/admin_components/adf-services/dreamfactory-services.js

```diff
@@ -364,6 +403,7 @@ angular.module('dfServices', ['ngRoute', 'dfUtility'])
                 scope.closeService = function () {
 
                     scope.$parent.currentEditService = null;
+                    top.window.location.hash = '#/apidocs';
```

Comment:
Is it really necessary to change URL?
would not page reload after we clear currentEdirService?

Reply:
Its not necessary, but if you go to an apidoc from a service (i.e you have the #service in the url), if you then click "back to service list" the url remains. And then. if you click another api doc, it loads up fine, but the url also remains.

Functionality is fine, it just looks a bit strange so i thought it would be better to clear it

Reply:
good catch!